features/aliases.feature

```diff
@@ -126,26 +126,102 @@ Feature: Create shortcuts to specific WordPress installs
     When I run `wp eval --skip-wordpress 'echo realpath( getenv( "RUN_DIR" ) );'`
     Then save STDOUT as {TEST_DIR}
 
-    When I run `wp cli alias`
+    When I run `wp cli alias list`
     Then STDOUT should be YAML containing:
       """
       @all: Run command against every registered alias.
       @foo:
         path: {TEST_DIR}/foo
       """
 
-    When I run `wp cli aliases`
+    When I run `wp cli alias list --format=json`
+    Then STDOUT should be JSON containing:
+      """
+      {"@all":"Run command against every registered alias.","@foo":{"path":"{TEST_DIR}/foo"}}
+      """
+
+  Scenario: Get alias information
+    Given an empty directory
+    And a wp-cli.yml file:
+      """
+      @foo:
+        ssh: user@host:/path/to/wordpress
+      """
+
+
+    When I run `wp cli alias get @foo`
+    Then STDOUT should be YAML containing:
+      """
+      ssh: user@host:/path/to/wordpress
+      """
+
+    When I try `wp cli alias get @someotherfoo`
+    Then STDERR should contain:
+      """
+      Error: No alias found with key '@rtmew'.
+      """
+
+  Scenario: Add an alias
+    Given an empty directory
+    And a wp-cli.yml file:
+    """
+    @foo:
+      ssh: foo@bar:/path/to/wordpress
+    """
+
+    When I run `wp cli alias add dev user@hostname /path/to/wordpress`
+    Then STDOUT should be:
+       """
+       Success: Added '@dev' alias.
+       """
+    When I run `wp cli alias list`
     Then STDOUT should be YAML containing:
       """
       @all: Run command against every registered alias.
       @foo:
-        path: {TEST_DIR}/foo
+        ssh: foo@bar:/path/to/wordpress
+      @dev
+        ssh: user@hostname:/path/to/wordpress
       """
 
-    When I run `wp cli alias --format=json`
-    Then STDOUT should be JSON containing:
+  Scenario: Delete an alias
+    Given an empty directory
+    And a wp-cli.yml file:
+    """
+    @foo:
+      ssh: foo@bar /path/to/wordpress
+    @test:
+      ssh: test@host /path/to/wordpress
+    """
+
+    When I run `wp cli alias delete @test`
+    Then STDOUT should be:
       """
-      {"@all":"Run command against every registered alias.","@foo":{"path":"{TEST_DIR}/foo"}}
+       Success: Deleted '@test' alias.
```

Comment:
I'd also verify this with `wp cli alias list`, just like in the "Add an alias" scenario above.